src/pages/home/report/ReportActionCompose.js

```diff
@@ -297,6 +340,35 @@ class ReportActionCompose extends React.Component {
                             </>
                         )}
                     </AttachmentModal>
+                    <Popover
+                        isVisible={this.state.isEmojiPickerVisible}
+                        onClose={this.hideEmojiPicker}
+                        hideModalContentWhileAnimating
+                        animationInTiming={1}
+                        animationOutTiming={1}
+                        anchorPosition={{
+                            top: this.state.emojiPopoverAnchorPosition.vertical - 360,
```

Comment:
Can we use constants for the emoji container width and height, then reference those constants both from the style file and here? I think that will make it clearer what's going on with the `anchorPosition` here